tasks/bake.js

```diff
@@ -27,7 +27,6 @@ module.exports = function( grunt ) {
 			basePath: "",
 			transforms: {},
 			parsePattern: /\{\{\s*([^\}]+)\s*\}\}/g,
-			transformGutter: "|",
```

Comment:
Good point. Let's do so. If we want to enable developers to be able to change the gutter, we can put the regular expression `transformRegex` (and `paramsRegex`) into options. But options grow... So I would say, warning: yes, regex as option: no. But decide what you prefer.

Yeah, as often one could never have enough information, would be great to know which functions are used and which is are just decoration. But we probably only notice afterwars, when we get complains about removed features :unamused:

Regarding the versioning... puh, no clue. Sounds right that cuts in the API should lead major version bump. But as it's *only* a configuration and also just recently introduced it think it's okay to bump minor version